README.md

```diff
@@ -11,9 +15,8 @@ used at the Biohub and in the broader imaging community.
 - OME-Zarr ([OME-NGFF v0.4](https://ngff.openmicroscopy.org/0.4/))
 - Micro-Manager TIFF sequence, OME-TIFF (MMStack), and NDTiff datasets
 - Custom data formats generated by Biohub microscopes
-  - Supported: Falcon (PTI), Dorado (ClearControl)
-  - WIP: Mantis, Dragonfly
-  - TBA: DaXi
+  - Supported: Falcon (PTI), Dorado (ClearControl), Dragonfly (OpenCell OME-TIFF)
+  - WIP: Mantis, DaXi
```

Comment:
I think mantis is already supported, no? Are there other features we've planned to add before considering it "supported"?

Reply:
I was thinking of #142 and forgot that it has been solved. Moving this to 'supported'.